src/collections/_documentation/data-management/fingerprint-database-connection-example/javascript.md

```diff
@@ -0,0 +1,20 @@
+```javascript
+class DatabaseConnectionError extends Error {}
+
+Sentry.init({
+  ...,
+  beforeSend: (event, hint) => {
+    const exception = hint.originalException;
+
+    if (!(exception instanceof Error)) {
```

Comment:
This block can be removed as it wont have any effect now (we return `event` at the very bottom already.

Reply:
Oh, good point.

Done.